components/match2/wikis/overwatch/match_group_input_custom.lua

```diff
@@ -271,15 +271,13 @@ function matchFunctions.getVodStuff(match)
 
 	match.links = {}
 	local links = match.links
-	if match.esl then links.esl = 'https://play.eslgaming.com/match/' .. match.esl end
```

Comment:
why remove this?
the site is still functional

Reply:
I couldn't find any use of it on the wiki

Reply:
alright